miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -221,6 +229,13 @@ int main(int argc, char *argv[])
                   "0: Use single metric\n\t"
                   "1: Shape + space-dependent size given analytically\n\t"
                   "2: Shape + adapted size given discretely; shared target");
+   args.AddOption(&hradaptivity, "-hr", "--hr-adaptivity", "-no-hr",
+                  "--no-hr-adaptivity",
+                  "Enable hr-adaptivity.");
+   args.AddOption(&amr_metric_id, "-hmid", "--h-metric",
+                  "same options as metric_id");
+   args.AddOption(&hessiantype, "-ht", "--Hessian type for hr-adaptivity examples",
+                  "1-3");
```

Comment:
Expain the options (don't go into details, just something general that corresponds to the paper tests).